bin/packages/build.js

```diff
@@ -164,9 +164,7 @@ function buildScssFile( styleFile ) {
 function buildJsFileFor( file, silent, environment ) {
 	const buildDir = BUILD_DIR[ environment ];
 	const destPath = getBuildPath( file, buildDir );
-	const babelOptions = getBabelConfig( environment );
-	babelOptions.sourceMaps = true;
```

Comment:
Moved this within the `getBabelConfig` function for coherence.

Reply:
It's fine, we just need to ensure it gets synced with https://github.com/WordPress/gutenberg/pull/14409 :)

Reply:
Rebased from master to get this in.